config/locales/en.yml

```diff
@@ -1,18 +1,37 @@
 ---
 en:
-  bank-holidays: bank-holidays
+  ab_testing:
+    explanation: We sometimes test different versions of pages with different users
+      to see which work better. This page shows us if the process is working.
+    version: There are 2 versions of this page. You are viewing the
+    visit_govuk: Visit the <a href="/" class="govuk-link">GOV.UK homepage</a> to find
+      government services and information.
+  and: and
+  bank-holidays: 
```

Comment:
Looks like we've got a `bank-holidays` and `bank_holidays` key, I think we should stick with the underscore version.

Reply:
I removed it from the English file but it got added back when I ran the command to add missing keys because the Welsh file had that key, I can delete it...